### DISCUSSION TITLE
Games that require Epic verification?

### DISCUSSION THREAD

**Discussion by clemon79** [Heroic-Games-Launcher/HeroicGamesLauncher#1201] [Q&A]
Thinking this might have more to do with Heroic than the Steam Deck so asking here first. (Plus Reddit is down right now. :))
 
So I’m trying to get Black Widow and Centipede Recharged to run in the normal Gaming Mode on my Steam Deck. I can get them to launch in Heroic in Desktop Mode, but even though I am logged into Heroic they both pop a browser window during launch that asks me to authenticate to my Epic account. Regardless if I Allow or Deny this access, the game launches, but I do have to do one or the other; just closing the browser window makes the game stick on the Loading screen.

So I’m trying to figure out if there is a way to configure this in Heroic to dismiss this screen for me, or pop it to the forefront so I can deal with it. Launching it in a full-screen window might be my best answer here but I still don’t like it (and I’m not sure it’s the fix). Anyone have an idea?

**Comment by flavioislima** [comment]
Hmm, we don't know why this happens and how to avoid this unfortunately since this might be an internal gaming function.
Maybe try to launch the game as offline to see if this happens.
Most of them when we click on allow, it never asks anymore. Not sure why this one is doing it.

**Reply by clemon79** [reply]
Yeah, tried that, no go. What I should (and will) do is first see if this thing behaves the same way in the actual Epic client on an actual Windows machine. If it does, well, whaddaya do. I worked at Atari at one time and I can’t say it surprises me too much. :) If it *doesn’t*, maybe registering it on the actual Epic client will flip some kind of flag that gets it to quiet down on Heroic?
